woocommerce/woocommerce-ios
# Issue/4548 Connect card reader UI

Closes: #4548


### Description


The PR:
* Changes the UI of "search" reader screen to follow [the design](vaw3DvewbUwuQnPXpdNGGH-fi-38%3A7368)
* UI is implemented with swiftUI
* Small fixes to the UI
* Reuses "learn more" section 

### Testing instructions

* Open settings
* Click in-person payments
* Click Manage card readers
* Notice the improved UI

### Screenshots


https://user-images.githubusercontent.com/4923871/148056653-7619e587-4d4f-4d4e-b578-82fbea3da2c8.mov

---
